### DISCUSSION TITLE
lower the alpha channel of the colored area?

### DISCUSSION THREAD

**Discussion by alexlezam** [ImageMagick/ImageMagick#7346] [Help]
### ImageMagick version

7.0.0.0

### Operating system, version and so on

.net

### Description

I am in C#, I have some images with white color on a transparent background.  example image
![jj](https://github.com/ImageMagick/ImageMagick/assets/107267893/953afc0c-34f8-49a8-baed-6e1f6cdf13ad)

I already know how to change the color, more like coloring on the image, with the code:
imagen.Colorize(color,percentage);

My question is how can I lower the alpha channel to the colored area so that it has some transparency?

**Comment by snibgo** [comment]
I don't use C#, but I expect it would be something like this:
```
image.Evaluate(Channels.Alpha, EvaluateOperator.Multiply, 0.75);
```

**Reply by alexlezam** [reply]
thanks, thnaks, thanks
